{
  try {
    _persistence.countByC_S(RandomTestUtil.nextLong(),RandomTestUtil.nextInt());
    _persistence.countByC_S(0L,0);
  }
 catch (  Exception e) {
    Assert.fail(e.getMessage());
  }
}
